src/fairness/weighted_tree/test/load_accounting_db.cpp

```diff
@@ -0,0 +1,331 @@
+/*****************************************************************************\
+ *  Copyright (c) 2020 Lawrence Livermore National Security, LLC.  Produced at
+ *  the Lawrence Livermore National Laboratory (cf, AUTHORS, DISCLAIMER.LLNS).
+ *  LLNL-CODE-658032 All rights reserved.
+ *
+ *  This file is part of the Flux resource manager framework.
+ *  For details, see https://github.com/flux-framework.
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the Free
+ *  Software Foundation; either version 2 of the license, or (at your option)
+ *  any later version.
+ *
+ *  Flux is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the IMPLIED WARRANTY OF MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE.  See the terms and conditions of the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+ *  See also:  http://www.gnu.org/licenses/
+\*****************************************************************************/
+
+#include <iostream>
+#include <fstream>
+#include <sqlite3.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <vector>
+#include <tuple>
+#include <deque>
+
+#include "src/fairness/weighted_tree/test/weighted_tree_load.hpp"
+
+using namespace Flux::accounting;
+
+// global deque of tuples to store accounting database records
+std::deque
+  <std::tuple<std::string, std::string, std::string, std::string, std::string>>
+  accounting_db;
+
+int get_sub_banks(
+  sqlite3* DB,
+  const std::string& bank_name,
+  int level,
+  sqlite3_stmt *b_select_shares_stmt,
+  sqlite3_stmt *b_select_sub_banks_stmt,
+  sqlite3_stmt *b_select_associations_stmt
+) {
```

Comment:
For readability and making this consistent with other C or C++ codes we have, the following would be better, I think. (Notice the leading whitespace before the open parenthesis as well).

```suggestion
int get_sub_banks (sqlite3* DB,
                   const std::string& bank_name,
                   int level,
                   sqlite3_stmt *b_select_shares_stmt,
                   sqlite3_stmt *b_select_sub_banks_stmt,
                   sqlite3_stmt *b_select_associations_stmt)
{
```